sdk/keyvault/Directory.Build.props

```diff
@@ -1,6 +1,6 @@
 ﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup Condition="$(MSBuildProjectName.EndsWith('.Samples'))">
-    <IsTestProject>true</IsTestProject>
+    <IsSampleProject>true</IsSampleProject>
```

Comment:
Do not change this. It breaks all the other samples under sdk/keyvault.

Reply:
Hi, @heaths I pushed a commit to avoid that config impacting MGMT SDK, could you review it again thanks.